pkg/store.go

```diff
@@ -25,9 +57,9 @@ type Store interface {
 	// Unreserved means not already being used in a pending transaction.
 	GetAllUnreservedUTXOs(account Address) ([]UTXO, error)
 
-	// Commit a list of Update structs transactionally.
-	// This interface suports both SQL and Object stores (with conditional puts)
-	Commit(updates []any) error
+	// What it says on the tin. We should consider
+	// adding this to Store as a fast-path
```

Comment:
I think every update, such as new UTXOs / verified txns, will always be part of a transaction e.g. from ChainFollower.